docs/developers/container-insights-ecs-prometheus.md

```diff
@@ -0,0 +1,173 @@
+# Container Insight ECS Prometheus
```

Comment:
can add a link to this user guide in main README page?

Reply:
I think we don't have a section for dev doc in README, I could add new section though.

Reply:
See latest README